zeppelin-web/src/test/java/org/apache/zeppelin/WebDriverManager.java

```diff
@@ -146,37 +151,48 @@ public Boolean apply(WebDriver d) {
     return driver;
   }
 
-  private static void downloadFireBug(int firefoxVersion, String tempPath) {
-    String firebugUrlString = null;
-    if (firefoxVersion < 23)
-      firebugUrlString = "http://getfirebug.com/releases/firebug/1.11/firebug-1.11.4.xpi";
-    else if (firefoxVersion >= 23 && firefoxVersion < 30)
-      firebugUrlString = "http://getfirebug.com/releases/firebug/1.12/firebug-1.12.8.xpi";
-    else if (firefoxVersion >= 30 && firefoxVersion < 33)
-      firebugUrlString = "http://getfirebug.com/releases/firebug/2.0/firebug-2.0.7.xpi";
-    else if (firefoxVersion >= 33)
-      firebugUrlString = "http://getfirebug.com/releases/firebug/2.0/firebug-2.0.17.xpi";
+  public static void downloadGeekoDriver(int firefoxVersion, String tempPath) {
+    String geekoDriverUrlString =
```

Comment:
@felixcheung  is this OK? I've seen something like https://github.com/Ardesco/selenium-standalone-server-plugin/blob/master/src/main/resources/RepositoryMap.xml but even that looks bit ugly.